app/views/users/verify_account/index.html.slim

```diff
@@ -6,6 +6,9 @@ p.mt-tiny.mb0 = t('forms.verify_profile.instructions')
 = simple_form_for(@verify_account_form, url: verify_account_path,
   html: { autocomplete: 'off', method: :post, role: 'form' }) do |f|
   = f.error :base
-  = f.input :otp, required: true, label: t('forms.verify_profile.name'), wrapper: :inline_form do
+  = f.input :otp, required: true, label: t('forms.verify_profile.name'), wrapper_html: { class: 'mb3' }, wrapper: :inline_form do
```

Comment:
To appease rubocop this line needs a line return probably right before `wrapper_html` so it doesn't exceed the 100 char line limit.